portstyle.css

```diff
@@ -28,23 +32,20 @@ body {
 }
 
 .navigationBar a {
-	margin: 20px auto;
-	font-size: 1.5rem;
+	margin: 20px 2%;
 	background: none;
 	border: none;
 	text-decoration: none;
 }
 
-
 .mainImage {
 	display: flex;
 	justify-content: center;
 	width: 100%;
 }
 
-
 .backgroundImage {
-	background-image: url("https://images.unsplash.com/photo-1592750088063-96b411cd5895?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1950&q=80");
+	background-image: url("file:///Users/academy/Documents/photo-1592750088063-96b411cd5895.htm");
```

Comment:
this needs to be a relative url, put the image in the same folder as your html and then just link the image file name

Reply:
Done for both.